com.pot_app.pot.yml

```diff
@@ -7,13 +7,11 @@ rename-icon: pot
 rename-desktop-file: pot.desktop
 
 finish-args:
-  - --socket=wayland
-  - --socket=fallback-x11
+  - --socket=x11
   - --socket=pulseaudio
   - --share=ipc
   - --share=network
   - --device=dri
-  - --env=WEBKIT_DISABLE_DMABUF_RENDERER=1
```

Comment:
The problem I mentioned before is an Nvidia problem and has nothing to do with Wayland. Even under X11, using Nvidia's private driver will cause a segfault.